### DISCUSSION TITLE
[closed][blueprint] label_short is almost as long as label_long, but should be limited to 14 chars

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#118] [Team Posts]
@dzerbino label_long and label_short are fetched directly from the trackhub fields. label_short is used by the browsers, I think, so I assumed this was prefilled for all the tracks in trackhub, and was mosty a matter of copy-paste. However, for BLUEPRINT, the short_label values are not short. Having two labels are thus quite pointless. 

Should we remove the label_short, or are they present in their short form in the existing metadata?

**Comment by dzerbino** [comment]
Alas, those are the only short labels at hand.

**Comment by sveinugu** [comment]
@dzerbino This does not make sense to me. I just crawled the original BLUEPRINT trackhub, and the tracks are denoted with long labels like "007MCL BS-Seq CPG_methylation_calls Mantle Cell Lymphoma signal from CNAG", with short labels like "007MCL.BS.Meth.MCLA".

**Comment by dzerbino** [comment]
Sorry, what's the problem? One is a short version of the other, is it not?

**Comment by sveinugu** [comment]
You can check in TrackFind (https://trackfind-dev.gtrack.no): Select tab "Blueprint_Hub_20160816 (...)" and look under configuration

**Comment by sveinugu** [comment]
Yes, but if I understood you correctly, you said that there were no short labels at hand?

**Comment by dzerbino** [comment]
Ah right, the issue is that the Blueprint metadata is splintered across multiple instances, I was looking in the central database only.

**Comment by sveinugu** [comment]
Just waiting to validate this change in the latest BLUEPRINT.

**Comment by sveinugu** [comment]
Validated. Many titles are way longer than 17 chars (changed from 14), though... However, this is a copy of the UCSC standard for the field, which it seems BLUEPRINT is anyway breaking.  Anyhow: the current char limit in FAIRtracks is a suggestion, so everything good. Closing this.
